examples/nuxt/nuxt.config.js

```diff
@@ -28,31 +28,18 @@ export default {
   // Modules for dev and build (recommended) (https://go.nuxtjs.dev/config-modules)
   buildModules: [
     // https://go.nuxtjs.dev/eslint
-    '@nuxtjs/eslint-module'
+    '@nuxtjs/eslint-module',
+    // 'vue-svg-loader/nuxt' // in a real project
+    '@/modules/vue-svg-inline-loader' // for this demo
```

Comment:
This approach is wrong. In this example you should resolve this new module directly from loader (something like `path.join(__dirname, "../../../nuxt/index.js")`) instead of creating duplicate code, that will need to stay in sync.

Reply:
True, I tried without the path.join and it didn't work so I made this workaround to test the plugin.